server/views/components/meta-unit/meta-unit.njk

```diff
@@ -0,0 +1,35 @@
+<div>
+  <{{model.headingLevel or 'h2'}} class="
+    {{ {s:'HNM5' , m:'HNM4', l:'HNM4'} | fontClasses }}
+    {{ {s:0, m:0, l:0, xl: 0} | spacingClasses({margin: ['top']}) }}
+    {{ {s:1, m:1, l:1, xl: 1} | spacingClasses({margin: ['bottom']}) }}">
+   {{ model.headingText }}
+  </{{model.headingLevel or 'h2'}}>
+  {% if model.content | isString %}
+    <p class="{{ {s:'HNL5' , m:'HNL4', l:'HNL4', xl:'HNL4'} | fontClasses }} {{ {s:1, m:1, l:1, xl: 1} | spacingClasses({margin: ['bottom']}) }}">{{model.content}}</p>
+  {% else %}
+    {% for item in model.content %}
+      {% if item.datetime %}
+        {% if loop.first %}
+          <p class="{{ {s:'HNL5' , m:'HNL4', l:'HNL4', xl:'HNL4'} | fontClasses }} {{ {s:1, m:1, l:1, xl: 1} | spacingClasses({margin: ['bottom']}) }}">
+        {% endif %}
+            <time datetime="{{ item.datetime }} ">{{ item.text }}</time>{{ ' - ' if not loop.last }}
+        {% if loop.last %}
+          </p>
+        {% endif %}
+      {% else %}
+          {% if loop.first %}
+          <ul class="plain-list {{ {s:1, m:1, l:1, xl: 1} | spacingClasses({margin: ['bottom']}) }}">
+          {% endif %}
+            <li class="inline"><a class="plain-link font--elf-green font-hover--mint {{ {s:'HNM5' , m:'HNM4', l:'HNM4', xl: 'HNM4'} | fontClasses }}" href="{{ item.url }}">{{ item.text }}</a>{{ ', ' if not loop.last }}</li>
```

Comment:
`font--elf-green font-hover--mint` – almost certainly the best class names I'm going to see this week :1st_place_medal: